### DISCUSSION TITLE
Running YOLO on IMX8QM MEK board

### DISCUSSION THREAD

**Discussion by swathi0508** [ultralytics/yolov5#12744] [Q&A]
Hi, we are trying to setup a project which requires running YOLOv5s or YOLOv8s on NXP's IMX8QM MEK board. Could you please help us in finding answers to the following?
1) Does this board support running the yolo models?
2) What is the minimum RAM requirement to run the pre-trained YOLOv5s/v8s?
3) Can this model be run on Automotive Grade Linux (AGL) OS?

If anybody has tried running the YOLO model on IMX8QM board, could you also please let us know the possible challenges?

**Comment by swadhinsekhar** [comment]
I tried using the YOLO model on the i.MX8Q board and discovered that YOLO can be exported to a TFLite model, which is supported by the i.MX8 libraries. Then, the model need to train using the eIQ toolkit,  to fully compatible with the i.MX8.
